### DISCUSSION TITLE
There has no official share button in the GitHub repository.

### DISCUSSION THREAD

**Discussion by arafat13495** [community/community#15748] [Code Security] [answered]
But if you go to any kind of website or big company website you can see an official share button whereby clicking a person can share those website content. but when you see in GitHub there has no official share button to share the website or repository.
I want a share button that will only be in the public repository and show. And through this, a person can quickly and officially share on GitHub and GitHub can count on the number of shares that the repository has shared.

**Comment by proarafat** [comment, marked as answer]
plz contact on Github Support
